app/src/lib/stores/app-store.ts

```diff
@@ -2176,6 +2183,15 @@ export class AppStore extends TypedBaseStore<IAppState> {
       )
     })
 
+    if (eligibleBranches.length >= FastForwardBranchesThreshold) {
```

Comment:
> This PR skips the fast-forward work when there are more than 20 eligible local branches.

This is skipping when there are more than 19 branches.

Reply:
Oops, yeah, off by one. The message that will be logged should be correct at least.